### ISSUE TITLE
Support internal storage 

### ISSUE THREAD

**Issue by lukeswitz** [lukeswitz/haven#29]
Only works with SD cards.  Nexus, galaxy support needed.

**Comment by lukeswitz** [comment]
Issue #322

**Comment by libBletchley** [comment]
To add to this, one rationale for internal storage is given here: https://github.com/guardianproject/haven/issues/62#issuecomment-410992977.

That is, it could perhaps control for malicious owners if they don't have root and want to destroy evidence of wrongdoing.

**Comment by lukeswitz** [comment]
Correct, it needs to be 100% cloud based- the title needs to be modified to "support devices without internal storage" 

Thank you!
